config/locales/two_factor_authentication/en.yml

```diff
@@ -28,7 +28,7 @@ en:
       instructions: If you don’t have access to another device, keep your backup codes
         safe. If you lose your backup codes, you won’t be able to sign into
         %{app_name}.
-      new_backup_codes_html: I need new backup&nbsp;codes
```

Comment:
I think this was to prevent a visually un-pleasing line break

Reply:
Ah, good catch ([link](https://github.com/18F/identity-idp/pull/8597#discussion_r1248109522)). It seems to be applied inconsistently, the button above it does not have the same behavior 🤔 

![image](https://github.com/18F/identity-idp/assets/1430443/067accbd-fe69-4d1d-872c-174427f40705)
![image](https://github.com/18F/identity-idp/assets/1430443/a83582dd-143a-4eaf-bff0-84e70a45c226)

Reply:
Maybe we can add the `&nbsp;` in the other one if needed, was there a test failure that lead to removing it here?

Reply:
https://github.com/18F/identity-idp/blob/a76b1464584831f75450a8bcd923309b908410c4/spec/features/two_factor_authentication/multiple_mfa_sign_up_spec.rb#L271 was the failing test

Reply:
maybe we just do a manual `gsub('&nbsp;', ' ')` on it to get the test to pass?

Reply:
good call, done